lib/portlayer/storage/storage.go

```diff
@@ -150,6 +159,9 @@ type DataSource interface {
 	//     nfs volume:  		 XDR-client
 	//     via guesttools:  	 toolbox client
 	Source() interface{}
+
+	// Given a filter spec
+	Stat(op trace.Operation, spec *archive.FilterSpec) (*FileStat, error)
```

Comment:
Comment does not conform to Godoc standard. Should be:
`Stat returns a minimal set of file stats for files matched by the spec`